### ISSUE TITLE
Adapting Text: Letter spacing

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-activity#478]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/wcag21/issues/390

**Comment by r12a** [comment]
Andrew: May also effect ligation, eg. fi (ie. with OpenType features)
Bert: Add proportional ratios, rather than absolute values.
r12a: May also have tracking set in CSS which should be respected, rather than just using font size metric.

**Comment by r12a** [comment]
I think we should close this because their repo is now archived, and the moment has passed.
